src/main.rs

```diff
@@ -48,11 +48,27 @@ fn warn_for_unsafe_execution() {
     warn!("WARNING: THIS tofnd BINARY IS NOT SAFE: SAFE PRIMES ARE NOT USED BECAUSE '--unsafe' FLAG IS ENABLED.  USE '--unsafe' FLAG ONLY FOR TESTING.");
 }
 
+/// prints the current tofnd version stated in cargo.toml
+fn print_version() -> TofndResult<()> {
+    Ok(println!(
+        "{} v{}",
+        std::env::var("CARGO_PKG_NAME")?,
+        std::env::var("CARGO_PKG_VERSION")?
+    ))
+}
+
 /// worker_threads defaults to the number of cpus on the system
 /// https://docs.rs/tokio/1.2.0/tokio/attr.main.html#multi-threaded-runtime
 #[tokio::main(flavor = "multi_thread")]
 async fn main() -> TofndResult<()> {
-    let cfg = parse_args()?;
+    print_version()?;
```

Comment:
I don't think we should print the version everytime (similar to how axelard didn't print it for every command we provide).

Reply:
that's true about axelard, but tofnd should start only once 🤔

Reply:
We do mnemonic commands, create etc. that aren't launching a service. So, it might be a bit weird to display version for those. And we might add more commands in the future?

Reply:
It will surely be weird if users need to spam tofnd commands and the version is always displayed. But until now we only expect to run it at most twice (setup and init). At any case, since displaying the version is not dependent on the actual service (so an existing tofnd instance won't block the version checking), users can execute `tofnd --version` at a separate terminal if needed, so I am fine with only displaying the version when the arg is given.